src/page/StoreRegistration/component/TimePicker/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState, useRef, useEffect } from 'react';
+import styles from './TimePicker.module.scss';
+
+const hours: number[] = [];
+const minutes: number[] = [];
+
+for (let i = 0; i < 24; i += 1) {
+  hours.push(i);
+}
+
+for (let i = 0; i < 60; i += 1) {
+  minutes.push(i);
+}
```

Comment:
```suggestion
const hours: number[] = Array.from({ length: 24 }, (_, i) => i);
const minutes: number[] = Array.from({ length: 60 }, (_, i) => i);
```
쓸데없이 길어보이니 하나로 보는게 좋아보입니다.

Reply:
for 반복문을 버릇처럼 쓰다보니 위와 같이 작성한 거 같습니다.
좀 더 코드를 줄이기 위해 고민하고 작성하도록 하겠습니다! 좋은 리뷰 감사합니다 1dab71d6d74c25394f622b07351418a35a707810